src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/ECDsa.cs

```diff
@@ -8,6 +8,20 @@ namespace System.Security.Cryptography
 {
     public abstract class ECDsa : AsymmetricAlgorithm
     {
+        public ECDsa() { }
+
+        public ECDsa(ECNamedCurve curve)
+        {
+            if (curve.Oid != null)
+            {
+                _curve = new Oid(curve.Oid.Value, curve.Oid.FriendlyName);
+            }
+        }
+
+        public abstract ECParameters ExportNamedCurveParameters(bool includePrivateParameters);
```

Comment:
@weshaggard since there is more than one nuget package involved, what is the best approach so the build doesn't break? I could
1) Add the virtual methods first as non-abstract in the base package (and wait for a build)
2) Add the overrides\implementation in the consuming package(s)
3) Go back to the base package and make the methods abstract

I have been using project-to-project references during development, but I am leery of checking in that way even though it would be temporary.